src/task/file_hashes.rs

```diff
@@ -111,7 +111,10 @@ impl FileHashes {
             .git_ignore(false)
             .git_global(false)
             .git_exclude(false)
-            .follow_links(true)
+            // Turn this back off as it can cause issues with symlinks:
+            // https://github.com/prefix-dev/pixi/issues/2196
+            // TODO: The current idea is to completely reimplement this without the `ignore` crate.
+            // .follow_links(true)
```

Comment:
This change shouldn't show up here I think :)

Reply:
There is a weird merge going on here.

Reply:
Non existent anymore